### PR TITLE
ProblemPage shows problem.message

### PR DESCRIPTION
**Pull request by T-Davis**
Part of [Error Handling UC 1 ](https://github.com/enspyrco/crowdleague/issues/274)

### CODE REVIEW

File: lib/widgets/shared/problem_page.dart [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import 'package:crowdleague/models/problems/problem_base.dart';
+import 'package:flutter/material.dart';
+
+class ProblemPage extends StatelessWidget {
```
Comment (nickmeinhold) [inline lib/widgets/shared/problem_page.dart:4]: Any interest in dropping a really brief class description comment at the top?  Class just seems kinda naked without one :-)  I am aware of the hypocrisy but I look forward to being asked to put some clothes on my classes!

**Review by nickmeinhold — CHANGES_REQUESTED** [review]
Beautiful mate!  Happy to add a class description comment (per my review comment)?

**Review by tamari-gray — APPROVED** [review]
lgtm! so good to see some problem pr's!

**Review by nickmeinhold — APPROVED** [review]
Brilliant!  Thanks mate!

**Pull request merged**